server/src/test/java/org/elasticsearch/search/aggregations/metrics/CardinalityTests.java

```diff
@@ -20,6 +22,9 @@ protected final CardinalityAggregationBuilder createTestAggregatorBuilder() {
         if (randomBoolean()) {
             factory.missing("MISSING");
         }
+        if (true) {
```

Comment:
Probably should be `if (randomBoolean)`

Reply:
oops, yeah.  I forced it for testing, and forgot to unforce it.